src/main/java/com/exactpro/epfast/template/xml/TemplatesXml.java

```diff
@@ -0,0 +1,76 @@
+package com.exactpro.epfast.template.xml;
+
+import com.exactpro.epfast.template.Dictionary;
+import com.exactpro.epfast.template.Template;
+import com.exactpro.epfast.template.Templates;
+
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import java.util.List;
+
+@XmlRootElement(name = "templates", namespace = NamespaceProvider.XML_NAMESPACE)
+public class TemplatesXml implements Templates, NamespaceProvider {
+
+    private NamespaceProvider parentNsProvider;
+
+    private String applicationNs;
+
+    private String templateNs;
+
+    private Dictionary dictionary;
+
+    private List<Template> templates;
+
+    @Override
+    public String getApplicationNamespace() {
+        if (applicationNs != null) {
+            return applicationNs;
+        }
+        if (parentNsProvider != null) {
+            return parentNsProvider.getApplicationNamespace();
+        }
+        return XML_NAMESPACE;
+    }
+
+    @Override
+    public String getTemplateNamespace() {
+        if (templateNs != null) {
+            return templateNs;
+        }
+        if (parentNsProvider != null) {
+            return parentNsProvider.getTemplateNamespace();
+        }
+        return XML_NAMESPACE;
```

Comment:
It should be `Reference.DEFAULT_NAMESPACE` or something alike. I don't remember how it's named exactly. Ask Vakho if you can't find it.